modules/plotting/processors/dataframeexporter.cpp

```diff
@@ -94,9 +94,13 @@ void DataFrameExporter::exportNow() {
         LogWarn("File already exists: " << exportFile_);
         return;
     }
-    if (exportFile_.getSelectedExtension() == exportFile_.getNameFilters()[0]) {
+    // convert the fileEnding to all lower case, for string comparison a few lines down.
+    std::string fileEnding = filesystem::getFileExtension(exportFile_.get());
+    std::transform(fileEnding.begin(), fileEnding.end(), fileEnding.begin(), ::tolower);
```

Comment:
this line will generate a warning since ::tolower returns an int not a char

One can use iCaseCmp from stringconversion.h to do case insensitive compare